src/derive/curve.rs

```diff
@@ -225,10 +225,6 @@ macro_rules! new_curve_impl {
             () => {
                 paste::paste! {
 
-                #[allow(non_upper_case_globals)]
-                const [< $name _FLAT_BYTE_INDEX >]: usize =
-                    2 *$base::size() - 1;
-
```

Comment:
I think we should keep this `_FLAG_BYTE_INDEX` for consistency with `compressed` code.